new-nextjs-client/src/entities/app-user.ts

```diff
@@ -3,6 +3,7 @@ import { immerable } from 'immer';
 interface IAppUser {
   years: number[];
   addYear: (year: number) => void;
+  sortYear: () => void;
```

Comment:
Remove this and make it a private method. which will be only called in `addYear()`